gateway-js/tsconfig.test.json

```diff
@@ -3,7 +3,7 @@
   "files": [
     "./src/__tests__/testSetup.ts"
   ],
-  "include": ["**/__tests__/**/*", "**/__mocks__/**/*"],
+  "include": ["**/__tests__/**/*", "**/__mocks__/**/*", "src/**/*"],
```

Comment:
gateway test includes everything in src?

Reply:
Beat me to it [589cd63](https://github.com/apollographql/federation/pull/1757/commits/589cd63e227d8a15543c8c88c37e9b2f93c64b96)